doc/rules-apple.md

```diff
@@ -117,6 +117,8 @@ objc_library(
 )
 ```
 
+NOTE: This is only supported on bazel 6.0+
```

Comment:
Does this only apply to `apple_static_framework_import`? What about `apple_static_xcframework`?

Reply:
actually i think this one is fine, just not in our tests, because we generate one to tetst it